drv/ksz8463/src/lib.rs

```diff
@@ -19,6 +59,8 @@ enum Trace {
 }
 ringbuf!(Trace, 16, Trace::None);
 
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
Out of curiosity (but *actually* out of curiosity!), what do you you use these breaks to denote? In particular would be curious to know if you use this in your workflow by searching forwards or backwards for them?  And obviously feel free to ignore this question entirely. ;)

Reply:
It's purely visual – I use them to break up logically distinct-ish sections of the file, so I can scan through and see major chunks at a glance.